pyproject.toml

```diff
@@ -46,7 +46,8 @@ pytest-cov = "^4.0"
 optional = true
 
 [tool.poetry.group.typing.dependencies]
-mypy = ">=0.942"
+mypy = "^1.1"
+lxml = "^4.9"
```

Comment:
Can't comment on the exact line, but I was expecting a `typing-extensions` dependency to be added. I think it would be good to explicitly add the dependency.

Reply:
You are right, ofcourse. But is a regular dependency for py<3.11, it is in the source code.